simplejson/encoder.py

```diff
@@ -17,10 +17,7 @@ def _import_speedups():
 from .decoder import PosInf
 from .raw_json import RawJSON
 
-#ESCAPE = re.compile(ur'[\x00-\x1f\\"\b\f\n\r\t\u2028\u2029]')
-# This is required because u() will mangle the string and ur'' isn't valid
-# python3 syntax
-ESCAPE = re.compile(u'[\\x00-\\x1f\\\\"\\b\\f\\n\\r\\t\u2028\u2029]')
+ESCAPE = re.compile(r'[\x00-\x1f\\"\b\f\n\r\t]')
```

Comment:
Actually since `\b\f\n\r\t` are all in the range `\x00-\x1f` the pattern can be just `r'[\x00-\x1f\\"]'`.

Reply:
Remove also entries for 0x2028 and 0x2029 from ESCAPE_DCT.